salt/metalk8s/addons/prometheus-operator/deployed/dashboards.sls

```diff
@@ -0,0 +1,34 @@
+#!jinja | kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+
+{%- set dashboards = [
```

Comment:
We re saving data into an sls file, should we create a debt ticket for this to move it somewhere else more maintainable ?

Reply:
It's not really 'data' IMHO, more of an `import ... with context` or rather `include`... Don't think 'magically' deploying all `files/*.json` dashboards would be wise, because the transform function may not suit all possible dashboard inputs (in the MetalK8s 1.x `hack/fix-dashboard.py` or something script, there's a whole bunch of quirks).

So, not sure... What would you think would be 'more maintainable'?